src/util/applyMixins.ts

```diff
@@ -0,0 +1,7 @@
+export default function applyMixins(derivedCtor: any, baseCtors: any[]) {
+  baseCtors.forEach(baseCtor => {
+    Object.getOwnPropertyNames(baseCtor.prototype).forEach(name => {
+      derivedCtor.prototype[name] = baseCtor.prototype[name];
+    })
+  });
+}
```

Comment:
Can we convert these `forEach` loops to regular `for` loops? It's a micro optimization, but I'd like to avoid `forEach` in the library if possible.